src/api/project/CachingProjectLoader.ts

```diff
@@ -14,16 +14,16 @@
  * limitations under the License.
  */
 
-import { ProjectLoader, ProjectLoadingParameters, WithLoadedProject } from "./ProjectLoader";
+import { ProjectLoader, ProjectLoadingParameters, WithLoadedProject } from "../../spi/repo/ProjectLoader";
```

Comment:
why repo instead of project? I see corresponding things in api/project

Reply:
Agreed. I can change this